### ISSUE TITLE
Create a View From Storyboard Name

### ISSUE THREAD

**Issue by ClearlyAwesome** [EurekaCommunity/ViewRow#26]
I'm trying to figure out how to use a storyboard name instead of a NIB name and it just isn't clicking. Is it even possible to add a View Row using SB name?

**Comment by alldritt** [comment]
I don't believe this is possible.  UIStoryboard only provides APIs for loading viewControllers, not views.

https://developer.apple.com/documentation/uikit/uistoryboard

I suppose you could load a view controller from your storyboard, and try to display the associated view in a ViewRow, but that probably introduces as many problems as it solves.
